app/globals.css

```diff
@@ -35,6 +35,13 @@
     --radius: 0.5rem;
   }
  
+@font-face{
```

Comment:
as much as this is correct for most cases, it's not correct for next.js. Reference how we currently do fonts here: https://github.com/coursepolicy/policy-generator/blob/7be29c79985f275cae9ac95d37818e83ba290d4d/app/(main)/layout.tsx#L10